website2/docs/guides-ui-guide.md

```diff
@@ -38,6 +38,18 @@ listed below. A complete set of features can be found in following sections.
 
 #### Topologies Page
 
+Heron UI is a user interface that uses the Heron Tracker to display detailed, colorful visual representations of topologies, including the logical and physical plan for each topology. 
+
+Start the Heron tracker using `heron-tracker &` which uses default heron_tracker.yaml configuration file. It's a centralized gateway for cluster-wide information about topologies, including which topologies are running, being launched, being killed, etc. It exposes Json Restful endpoint and relies on Zookeeper nodes.
+
+Launc the Heron UI by the command:
+
+```bash
+heron-ui &
+```
+
+By default Heron UI will be started at `http://localhost:8888`
```

Comment:
I think by default the Heron UI is served at `localhost:8889` right?

Reply:
Oh ya Mistake. Updating it
Heron Tracker: 8888
Heron UI: 8889